dbt_sugar/core/connectors/postgres_connector.py

```diff
@@ -0,0 +1,55 @@
+"""
+Module Postgres connector.
+
+Module dependent of the base connector.
+"""
+from typing import List
+
+import psycopg2
+from base import BaseConnector
+
+
+class PostgresConnector(BaseConnector):
+    """
+    Connection class for Postgres.
+
+    Child class of base connector.
+    """
+
+    def __init__(self, user: str, password: str, database: str, host: str = "localhost") -> None:
+        """
+        Init method to instanciatee the credentials.
+
+        :param user: string with the user name.
+        :param password: string with the password.
+        :param database: string with the database name.
+        :param host: string with the host name.
+        """
+        self.connection_details = dict(
+            host=host,
+            user=user,
+            password=password,
+            database=database,
+        )
+
+    def generate_connection(self):
+        """
+        Method that creates the connection.
+
+        :return: psycopg2.connector.connection
+        """
+        return psycopg2.connect(**self.connection_details)
```

Comment:
I see that you're using two different pacakages to talk to dbs. 
We should be able to use `sqlalchemy` for both of these at least they both are supported. If you look in the tests you can see how I do it.

Snowflake ships a `snowflake-sqlalchemy` package which can be used to generate db URL or we can generate the string ourselves.